compiler/optimizer/GlobalValuePropagation.hpp

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2000, 2016
```

Comment:
No copyright headers should be updated to have the second year match the current year when the file is touched in general. We haven't always been as vigilant as we should, but for the VP files touched they should all have 2017 as the second year as part of this commit.